tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py

```diff
@@ -40,19 +40,27 @@ def test_ftu_skip_tour(self):
         ssid = self.testvars['wifi']['ssid']
         psk = self.testvars['wifi'].get('psk')
         keymanagement = self.testvars['wifi'].get('keyManagement')
+        share_data_default = True
```

Comment:
Okay, I don't see any reason why this constant should be used, then. Thank you for the explanation.